#!/usr/bin/env python

PACKAGE = "pure_pursuit"

from dynamic_reconfigure.parameter_generator_catkin import *
gen = ParameterGenerator()

gen.add("vel_linear_max", double_t, 0, "Value of maximum linear velocity", 0.3, 0, 2.0)

exit(gen.generate(PACKAGE, "PurePursuitConfig", "PurePursuit"))